<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>300</y>
      <width>371</width>
      <height>121</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background: none;</string>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="digitCount">
     <number>5</number>
    </property>
    <property name="value" stdset="0">
     <double>88888.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="setAlarmButton">
    <property name="geometry">
     <rect>
      <x>495</x>
      <y>12</y>
      <width>160</width>
      <height>105</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="alarmOnOffButton">
    <property name="geometry">
     <rect>
      <x>495</x>
      <y>122</y>
      <width>160</width>
      <height>105</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumberAlarm">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>300</y>
      <width>371</width>
      <height>121</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background: none;</string>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="digitCount">
     <number>5</number>
    </property>
    <property name="value" stdset="0">
     <double>88888.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="confirmAlarm">
    <property name="geometry">
     <rect>
      <x>634</x>
      <y>158</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="addMinuteButton">
    <property name="geometry">
     <rect>
      <x>492</x>
      <y>158</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="addHourButton">
    <property name="geometry">
     <rect>
      <x>361</x>
      <y>158</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="alarmOnLabel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>300</y>
      <width>89</width>
      <height>113</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
